### DISCUSSION TITLE
Welcome to xiu Discussions!

### DISCUSSION THREAD

**Discussion by harlanc** [harlanc/xiu#163] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GrumpyChubbyCat** [comment]
Hello, I'm a computer vision engineer and I often have the need to stream video. Typically the following pipline is used: computer vision applications stream to a media server, in rtsp format. From the media server, clients can view the streams via HLS/WebRTC. In this regard, will Xiu be able to convert the stream from rtsp to WebRTC?

**Reply by harlanc** [reply]
HI , now the RTSP stream can be converted to HTTPFLV/HLS/RTMP, and to WebRTC now is not supported.. https://www.rustxiu.com/docs/remuxers/introduction/

**Reply by GrumpyChubbyCat** [reply]
Thanks for the reply! Will support for RTSP to WebRTC conversion be added in the future?
